lib/node_modules/@stdlib/math/base/ops/imuldw/benchmark/benchmark.js

```diff
@@ -49,3 +49,27 @@ bench( pkg, function benchmark( b ) {
 	b.pass( 'benchmark finished' );
 	b.end();
 });
+
+bench( pkg+'::assign', function benchmark( b ) {
```

Comment:
```suggestion
bench( pkg+':assign', function benchmark( b ) {
```

Use only a single colon when describing a method.